strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/testing/storage/repository/RepositoryAttributes.java

```diff
@@ -40,4 +40,6 @@
 
     boolean checksumHeadersEnabled() default false;
 
+    boolean strictChecksum() default false;
```

Comment:
Could you please change this as well, like we discussed earlier to: `strictChecksumValidation`? (And could you please check, if there are any other places like that need to be changed?). Thanks! :)

Reply:
Done!